pkg/packagekit/updates.jsx

```diff
@@ -301,7 +314,7 @@ function UpdateHistory(props) {
 class ApplyUpdates extends React.Component {
     constructor() {
         super();
-        this.state = { percentage: 0, timeRemaining: null, curStatus: null, curPackage: null };
+        this.state = { percentage: 0, timeRemaining: null, actions: [] };
```

Comment:
Can you add a comment on how `actions` is filled?
Do you think it makes sense to use an object for each entry instead? `action[0]` and `action[1]` are a lot more difficult to read and debug than `action['packageInfo']`